src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -450,6 +450,19 @@
     <Field Name="Method" Type="MethodSymbol"/>
   </Node>
 
+  <!-- Represents the greatest raw metadata token value for any method definition in the current module. -->
```

Comment:
I am Ok with length. The reason I used MethodDef is because it makes it clearer that this is about metadata. Otherwise there is a question what index are we talking about, index in where.